test/jdk/java/lang/annotation/LoaderLeakTest.java

```diff
@@ -46,7 +77,7 @@ static void doTest(boolean readAnn) throws Exception {
         // URL[] path = { classes };
         // URLClassLoader loader = new URLClassLoader(path);
```

Comment:
please remove the commented out lines (L76-78)

Reply:
That indentation is intended to make the code more readable, at least I find it so. Changed now to more "traditional style".